addons/udes_security/views/no_desktop_access_template_views.xml

```diff
@@ -9,7 +9,7 @@
                         <div t-attf-class="col-12 col-md col-lg-12">
                             <h3>Access Denied</h3>
                             <p>You do not have access to the desktop, please contact your manager if you believe this is an error.</p>
-                            <p>
+                            <p t-if="reset_password_enabled">
```

Comment:
Dont understand how removing the t-if helps?

Reply:
No you are not missing i am missing, thought that was the t-if and removed as part of the changes :(.
Sorry